be/src/storage/lake/schema_change.cpp

```diff
@@ -136,6 +136,9 @@ Status ConvertedSchemaChange::init() {
     _read_params.fill_data_cache = false;
 
     ASSIGN_OR_RETURN(auto base_tablet_schema, _base_tablet->get_schema());
+    if (base_tablet_schema->keys_type() == KeysType::PRIMARY_KEYS) {
+        _read_params.sorted_by_keys_per_tablet = true;
+    }
```

Comment:
```suggestion
    _read_params.sorted_by_keys_per_tablet = true;
```
It seems that DUPLICATE KEY model will have the same problem.

Reply:
Done